README.md

```diff
@@ -209,22 +210,26 @@ The service can be started against a set of non-existent database. If no pre-exi
 4. Start the Admin project and this will configure the contained users' permissions via database migrations. 
    The other projects will then be able to be started, using their own contained users to connect to the databases. 
 
+### Setting up the storage emulator
+The Azurite Docker container can be started by one of the following methods:
+1. Indirectly by starting the admin via the start script
+```bash
+    pnpm start admin
+```
+2. Directly via the start script using pnpm start dataStorage
+```bash
+    pnpm start dataStorage
+```
+3. Directly via Docker Compose
+```bash
+    docker-compose up data-storage
+```
```

Comment:
Formatting around this is a little messy:

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/9917868/612858ad-3557-4757-8778-ccf10677650f)

Would expect it to look like this:

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/9917868/e1e4957f-eb78-4014-9741-45484870ece9)

- Code blocks don't align correctly with the list items 
- No need to start code block contents with an indentation - code should as far left as possible
- Could space out the list items so the raw Markdown reads nicer

Not going to quote any code directly as it won't output correctly in GitHub, but something like this would be nicer:

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/9917868/c1d70fee-ac12-4526-b17d-fd180ab17e16)